src/flownet/templates/observations.yamlobs.jinja2

```diff
@@ -5,7 +5,8 @@ smry:
    {% if loop.first and not loop.last: %}
  - key: WOPR:{{ well_name }}
    observations:
-   {%- elif not loop.first: %}
+   {%- endif -%} 
+   {%- if (kw.date.toordinal() > schedule.get_first_date().toordinal()) and (kw.date.toordinal() >= schedule.get_dates()[num_beginning_date-1].toordinal() )  and (kw.date.toordinal() <= schedule.get_dates()[num_end_date-1].toordinal()) %}
```

Comment:
Converting everything to ordinals is a bit messy. Can't you just keep working with pydates?

Reply:
I didn't find a way to compare "<=" or "<="between pydates. It appear that it just support < , > and ==. I'll read about it and find a better way.

Reply:
?

![image](https://user-images.githubusercontent.com/9119793/98529950-940c7480-227e-11eb-8297-fcc0f109b3f4.png)